Cargo.toml

```diff
@@ -25,25 +25,25 @@ categories = [
 crate-type = ["cdylib", "lib", "staticlib"]
 
 [dependencies]
-arti-client = { version = "0.5", default-features = false, features = [
+arti-client = { version = "0.7", default-features = false, features = [
```

Comment:
can you also bump the other deps while at it?

Reply:
They are bumped - `arti` is quite strange when it comes to versioning. If you look at

https://gitlab.torproject.org/tpo/core/arti/-/blob/main/crates/arti-client/Cargo.toml

you can see that all other dependencies from arti, `tor-*`, are all the same version as what I have here. I made an issue some time ago to complain that some patch-versions broke the dependencies. They said it shouldn't, but then did it again...

So we gotta live with it.

Reply:
hum, too bad, surely it'll be more stable as the first stable CLI has been released; ho well..

I meant the other deps, such as `postage` and `jni`, you can use [`cargo outdated -d1`](https://github.com/kbknapp/cargo-outdated) to know about availbe updates

Reply:
One more cargo tool :)

Unrelated question: why do the cargo-tools always propose to install with `--locked`?